### DISCUSSION TITLE
Slow hover animations

### DISCUSSION THREAD

**Discussion by vedeja** [likec4/likec4#600] [General]
I just discovered likec4 today and as a year-old user of C4 and Structurizr, I am very excited to find this. My first impresson is that it addresses many of the problems I've had, like limitiations in levels. However, the hover animations are extremely slow, I'd say to the point where the UI gets unusable. Any ideas what I can do?

**Comment by davydkov** [comment]
Hi @vedeja 

Thanks for your interest in the project.

I’ve received similar feedback from others about animations. It's a complicated issue, as some have reported smooth animations on old laptops and poor performance on high-end systems.

The good news is that I've been experimenting with another renderer, which should also address these issues.
The upcoming LikeC4 1.0 version will include this new renderer.

I am gonna publish pre-release quite soon.

**Reply by vedeja** [reply]
Thanks for confirming the issue, even though it's not consistent. I'll keep an eye out for that 1.0 pre-release.

**Reply by vedeja** [reply]
Hi again, @davydkov. I am seeing improvements in performance as I continue to refactor my model. Especially when splitting large quantities of elements into new groups, forming more levels. And also by nesting relationships inside their "source" element rather than spreading them all across the workspace. My workspace has about 400 elements, 500 relationships and is spread across 40 .c4 files.

**Reply by susliko** [reply]
Hi! 
Experiencing slow animations as well. Having a setting to disable them would be great
